npc/custom/warper.txt

```diff
@@ -3,7 +3,7 @@
 //===== By: ================================================== 
 //= Euphy
 //===== Current Version: =====================================
-//= 1.4b
+//= 1.5
```

Comment:
Would be nice to convert it to the new default header and getting rid of this and some other useless fields.